models/index.js

```diff
@@ -38,14 +35,11 @@ if (config.use_env_variable) {
   sequelize = new Sequelize(config.database, config.username, config.password, config);
 }
 
-fs.readdirSync(__dirname)
-  .filter(file => {
-    return file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js';
-  })
-  .forEach(file => {
-    const model = sequelize.import(path.join(__dirname, file));
-    db[model.name] = model;
-  });
+db.Case = require('./case')(sequelize, Sequelize.DataTypes);
+db.CaseAudit = require('./case-audit')(sequelize, Sequelize.DataTypes);
+db.Contact = require('./contact')(sequelize, Sequelize.DataTypes);
+db.CSAMReport = require('./csam-report')(sequelize, Sequelize.DataTypes);
+db.PostSurvey = require('./post-survey')(sequelize, Sequelize.DataTypes);
```

Comment:
Nice! I think this piece was the biggest issue in upgrading because `sequelize.import` was removed from version 6. Good to know we can easily require them.

One thing to notice is that with the current implementation we have to remember to change this file with each new model we create. Later on, we could try making this piece more dynamic.

Reply:
The linter was telling me off for trying to require them dynamically, and since there are quite a few other places where we have to remember to add a require / import to an index module I thought it was ok to do it here.

Besides, I'm hoping we are going to be removing more models than we're adding to Sequelize in the future :-P